docs/recitations/reci7-docker.md

```diff
@@ -0,0 +1,53 @@
+
+# Recitation 7:  Microservices and Docker
+
+## Prerequisites
+
+You should have downloaded Docker. If you haven't follow the installation instructions [here](https://docs.docker.com/get-docker/)
+ 
+## Setup Instructions (10 min): 
+
+Fork [this repo](https://github.com/CMU-313/s23-docker-recitation) and clone it.
+Also start the docker engine.
+ 
+## Overview
+
+During this recitation, students will create a simple FastAPI app, containerize it and deploy it.
+ 
+## Context
+
+Sadly, you still are unsure when your recitation time is AND who your TAs are. There exist a microservice that tells you which TA's are responsible for each section.
+
+Try it out using this link: [https://whos-my-ta.fly.dev/section_id/a](https://whos-my-ta.fly.dev/section_id/a)
+
+All you have to do is create another service that return the time of the recitation as well.
+
+The endpoint has to return a JSON object in the following form:
+```json
+{
+    "section": "section_name",
+    "start_time": "HH:MM",
+    "end_time": "HH:MM",
+    "ta": ["taName1", "taName2"]
+}
+```
+
+ 
+## Activity
+
+1. Implement the `section_info` endpoint according to specifications. Test it by running the app locally.
```

Comment:
Give them a little more guidance here. What file should the change? What's already in the file, what needs to change? Adding comments to `main.py` will also help.